tensorflow/core/api_def/base_api/api_def_Asin.pbtxt

```diff
@@ -1,4 +1,19 @@
 op {
   graph_op_name: "Asin"
-  summary: "Computes asin of x element-wise."
+  summary: "Computes the trignometric inverse sine of x element-wise."
+  description: <<END
+The `tf.math.asin` operation returns the inverse of `tf.math.sin`, such that
+if `y = tf.math.sin(x)` then, `x = tf.math.asin(y)`.
```

Comment:
sin isn't invertible, so this is true for the invertible range [-pi/2, pi/2]. That would be worth mentioning.